### PR TITLE
Contribution of Additional KennaV2 Command

### PR DESCRIPTION
**Pull request by rod-castillo**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by barchen1 — APPROVED** [review]

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 1]
```diff
@@ -514,7 +514,74 @@ script:
     description: Deletes tags from the specified asset.
     execution: false
     name: kenna-delete-tag
-  dockerimage: demisto/python3:3.8.6.14516
+  - arguments:
+    - default: false
+      description: Unique numerical ID of the connector
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:519]:
```suggestion
      description: Unique numerical ID of the connector.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 2]
```diff
@@ -514,7 +514,74 @@ script:
...
+      isArray: false
+      name: connector_id
+      required: true
+      secret: false
+    deprecated: false
+    description: Returns JSON data on all the runs of a given connector
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:525]:
```suggestion
    description: Returns JSON data on all the runs of a given connector.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 3]
```diff
@@ -514,7 +514,74 @@ script:
...
+    execution: false
+    name: kenna-get-connector-runs
+    outputs:
+    - contextPath: Kenna.ConnectorRunsList.ID
+      description: Connector Run ID
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:530]:
```suggestion
      description: Connector Run ID.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 4]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.StartTime
+      description: Connector Run Start Time
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:533]:
```suggestion
      description: Connector Run Start Time.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 5]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.EndTime
+      description: Connector Run End Time
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:536]:
```suggestion
      description: Connector Run End Time.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 6]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: string
+    - contextPath: Kenna.ConnectorRunsList.Success
+      description: Boolean value showing connector success
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:539]:
```suggestion
      description: Boolean value showing connector success.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 7]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: boolean
+    - contextPath: Kenna.ConnectorRunsList.TotalPayload
+      description: Total connector payloads
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:542]:
```suggestion
      description: Total connector payloads.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 8]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.ProcessedPayload
+      description: total payloads processed by connector
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:545]:
```suggestion
      description: Total payloads processed by the connector.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 9]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.FailedPayload
+      description: Total failed payloads
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:548]:
```suggestion
      description: Total failed payloads.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 10]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.ProcessedAssets
+      description: Count of processed assets
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:551]:
```suggestion
      description: Amount of processed assets.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 11]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.AssetsWithTagsReset
+      description: Count of assets with reset tags
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:554]:
```suggestion
      description: Amount of assets with reset tags.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 12]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.ProcessedScannerVulnerabilities
+      description: Count of processed scanners with vulnerabilities
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:557]:
```suggestion
      description: Amount of processed scanners with vulnerabilities.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 13]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.UpdatedScannerVulnerabilities
+      description: Count of updated scanners with vulnerabilities
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:560]:
```suggestion
      description: Amount of updated scanners with vulnerabilities.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 14]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.CreatedScannerVulnerabilities
+      description: Count of created scanners with vulnerabilities
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:563]:
```suggestion
      description: Amount of created scanners with vulnerabilities.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 15]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.ClosedScannerVulnerabilities
+      description: Count of closed scanners with vulnerabilities
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:566]:
```suggestion
      description: Amount of closed scanners with vulnerabilities.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 16]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.AutoclosedScannerVulnerabilities
+      description: Count of auto-closed scanners with vulnerabilities
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:569]:
```suggestion
      description: Amount of auto-closed scanners with vulnerabilities.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 17]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.ReopenedScannerVulnerabilities
+      description: Count of reopened scanners with vulnerabilities
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:572]:
```suggestion
      description: Amount of reopened scanners with vulnerabilities.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 18]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: number
+    - contextPath: Kenna.ConnectorRunsList.ClosedVulnerabilities
+      description: Count of closed vulnerabilities
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:575]:
```suggestion
      description: Amount of closed vulnerabilities.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 19]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.AutoclosedVulnerabilities
+      description: Count of auto-closed vulnerabilities
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:578]:
```suggestion
      description: Amount of auto-closed vulnerabilities.
```

File: Packs/Kenna/Integrations/KennaV2/KennaV2.yml [hunk 20]
```diff
@@ -514,7 +514,74 @@ script:
...
+      type: Number
+    - contextPath: Kenna.ConnectorRunsList.ReopenedVulnerabilities
+      description: Count of re-opened vulnerabilities
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/KennaV2.yml:581]:
```suggestion
      description: Amount of re-opened vulnerabilities.
```

File: Packs/Kenna/Integrations/KennaV2/README.md [hunk 21]
```diff
@@ -791,3 +791,81 @@ There is no context output for this command.
 
 ##### Human Readable Output
 Tag My test tag was deleted to asset {asset_id}
+
+
+### kenna-get-connector-runs
```
Comment (roysagi) [inline Packs/Kenna/Integrations/KennaV2/README.md:796]: Please change the doc with the notes I gave in the .yml

**Review by roysagi — COMMENTED** [review]
@barchen1 - Please address my comments and ping me.

**Pull request merged**